jiant/tasks/lib/boolq.py

```diff
@@ -91,13 +92,22 @@ def get_test_examples(self):
     @classmethod
     def _create_examples(cls, lines, set_type):
         examples = []
-        for (i, line) in enumerate(lines):
+        for line in lines:
             examples.append(
                 Example(
-                    guid="%s-%s" % (set_type, i),
+                    # NOTE: super_glue_format_preds() is dependent on this guid format.
+                    guid="%s-%s" % (set_type, line["idx"]),
```

Comment:
If it's not too much trouble, mind specifying that it's `BoolQTask.super_glue_format_preds`?

Reply:
Correspondingly for the other tasks.